styles/qr-code.styl

```diff
@@ -0,0 +1,3611 @@
+body
```

Comment:
don't add styles to html tags, Conflict will be there, instead please add classnames and add styles to it